Automattic/wp-calypso
# Add Featured image template toggle scaffolding

#### Proposed Changes

The proposed changes add scaffolding for the new Featured image template toggle:

![Markup on 2022-09-28 at 16:11:06](https://user-images.githubusercontent.com/25105483/192801740-aaa7e745-3013-45dd-8285-5a231cfac9a6.png)

During development, the setting will be behind the feature flag, available at `http://calypso.localhost:3000/settings/writing/`.

#### Testing Instructions



1. Make sure https://github.com/Automattic/wp-calypso/issues/68380 is already merged.
2. Navigate to `http://calypso.localhost:3000/settings/writing/`.
3. The Featured Image section should be available. It doesn't have any function at the moment. It is just a scaffold we will build on top of later.
4. Review console for any issues.

#### Pre-merge Checklist






Related to #68403
